docs/core/diagnostics/observability-with-otel.md

```diff
@@ -0,0 +1,429 @@
+
+## What is observability
+
+Observability in the context of a distributed system is the ability to monitor and analyze telemetry about the state of each component, to be able to observe changes in performance, and to diagnose why those changes occur. Unlike debugging, which is invasive and can affect the operation of the application, observability is intended to be transparent to the primary operation and have a small enough performance impact that it can be used continuously.
+
+Observability is commonly done using a combination of:
+
+- Logs - record individual operations, such as an incoming request, a failure in a specific component, or an order being placed.
+- Metrics - measuring counters and gauges such as number of completed requests, active requests, widgets that have been sold; or a histogram of the request latency.
+- Distributed tracing - track requests and activities across components in a distributed system so that you can see where time is spent and track down specific failures.
+
+Together, logs, metrics, and distributed tracing are known as the *3 pillars of observability*.
+
+Each pillar might include telemetry data from:
+
+- The .NET runtime, such as the garbage collector or JIT compiler.
+- Libraries, such as from Kestrel (the ASP.NET web server) and `HttpClient`.
+- Application-specific telemetry that's emitted by your code.
+
+## Observability approaches in .NET
+
+There are a few different ways to achieve observability in .NET applications:
+
+- Explicitly in code, by referencing and using a library such as OpenTelemetry. If you have access to the source code and can rebuild the app, then this is the most powerful and configurable mechanism.
+- Out-of-process using [EventPipe](./eventpipe.md). Tools such as [dotnet-monitor](./dotnet-monitor.md) can listen to logs and metrics and then process them without affecting any code.
+- [Using a startup hook](https://github.com/dotnet/runtime/blob/main/docs/design/features/host-startup-hook.md), assemblies can be injected into the process that can then collect instrumentation. An example of this approach is [OpenTelemetry .NET Automatic Instrumentation](https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/blob/main/docs/README.md).
+
+## What is OpenTelemetry
+
+[OpenTelemetry](https://opentelemetry.io/) (OTel) is a cross-platform, open standard for collecting and emitting telemetry data. OpenTelemetry includes:
+
+- [APIs](https://opentelemetry.io/docs/concepts/instrumentation/manual/) for libraries to use to emit telemetry
+- [Semantic conventions](https://github.com/open-telemetry/semantic-conventions) for metrics & traces
+- Standard for [exporter components](https://opentelemetry.io/docs/collector/)
```

Comment:
Exporters are in-process libraries that know how to format data for a specific backend. This link to OTel collector appears unrelated. I think you want: https://opentelemetry.io/docs/concepts/components/#exporters